x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/utils.ts

```diff
@@ -1006,3 +1007,23 @@ export const getMaxSignalsWarning = (): string => {
 export const getSuppressionMaxSignalsWarning = (): string => {
   return `This rule reached the maximum alert limit for the rule execution. Some alerts were not created or suppressed.`;
 };
+
+export const getDisabledActionsWarningText = ({
+  alertsCreated,
+  disabledActions,
+}: {
+  alertsCreated: boolean;
+  disabledActions: SanitizedRuleAction[];
+}) => {
+  const actionTypes = disabledActions.map((action) => action.actionTypeId);
```

Comment:
We should also deduplicate action types here to avoid messages like this:

<img width="1663" alt="Screenshot 2024-05-29 at 11 32 46" src="https://github.com/elastic/kibana/assets/92328789/5faaac09-59fc-42b9-b3e4-27d88b12823d">


I had 2 different connectors of the same type applied to a rule